proto/decentraland/sdk/components/ui_background.proto

```diff
@@ -5,7 +5,30 @@ import "decentraland/sdk/components/common/id.proto";
 option (common.ecs_component_id) = 1053;
 
 import "decentraland/common/colors.proto";
+import "decentraland/common/texture.proto";
+import "decentraland/common/border_rect.proto";
 
 message PBUiBackground {
-  optional decentraland.common.Color4 background_color = 1; // default=(0.0, 0.0, 0.0, 0.0)
-}
\ No newline at end of file
+  optional decentraland.common.Color4 color = 1; // default=(1.0, 1.0, 1.0, 1.0), pixel = color * sample2D(texture, uv)
```

Comment:
Can an UiBackground be at the same time a color and a texture?
If it can't, dismiss this comment.
 
If it can: this might be an `oneof` (as Material or Meshes) property. 
```
message PBUiBackground {
  message UiBackgroundTexture {
    decentraland.common.Texture texture = 1;
    BackgroundTextureMode background_texture_mode = 2;
    optional decentraland.common.BorderRect texture_slices = 4; // default=(1/3, 1/3, 1/3, 1/3)
    repeated float uvs = 5; // default=[0,0,0,1,1,0,1,0]: starting from bottom-left vertex clock-wise
  }
  oneof content {
    decentraland.common.Color4 color = 1;
    UiBackgroundTexture texture = 2;
  }
}
```

Reply:
Yes, it can be: it's the color that is multiplied with texture